src/ontoform/transformers/expressions_tissue.py

```diff
@@ -22,4 +22,4 @@ def transform(src: BinaryIO, dst: BinaryIO, format: SupportedFormats) -> None:
     output = pl.concat(tissue_list)
 
     # write the output to the destination
-    output.write_ndjson(dst)
+    write_dst(output, dst, SupportedFormats.PARQUET)
```

Comment:
```suggestion
    write_dst(output, dst, format)
```

So the writer decides depending on the argument passed. `SupportedFormats.PARQUET` is still the default.